contracts/contracts/lib/CrossMsgHelper.sol

```diff
@@ -199,4 +203,8 @@ library CrossMsgHelper {
 
         return true;
     }
+
+    function validateCrossMessage(IpcEnvelope memory crossMsg) internal view returns (CrossMessageValidationOutcome)  {
```

Comment:
caller why not just invoke `LibGateway` directly instead, it's internal anyways.

Reply:
Raul has suggested a nice syntax sugar update. So I have added it after his comment.